crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1704,7 +1706,30 @@ impl Room {
         self.send_state_event(RoomPowerLevelsEventContent::from(power_levels)).await
     }
 
-    async fn get_room_power_levels(&self) -> Result<RoomPowerLevels> {
+    /// Creates a set of power level changes based on the current power levels
```

Comment:
I think, it might be worth to say that it's the “constructor of `RoomPowerLevelChanges`”.

Reply:
Ok, I'm removed this entirely (see below).